tests/acceptance/01_vars/01_basic/sys_policy_version.cf

```diff
@@ -0,0 +1,17 @@
+body common control
+{
+  bundlesequence => { "check" };
+  version => "1.2.3";
+}
+bundle agent check
+{
+  classes:
+    "ok"
+      expression => strcmp("$(sys.policy_version)", "$(default:control_common.version)");
```

Comment:
This resolves to `"1.2.3"`? Maybe you could also test that it matches this string?